uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -37,7 +37,7 @@ define([
       this.requestDayTypes = [];
       this.selectedAbsenceType = {};
       this.period = {};
-      this.statusLabel = '';
+      this.statusBeforeEdit = {};
```

Comment:
it is before user starts to edit and it does not track the status of status upon edit , hence referred to before edit